test/common.js

```diff
@@ -402,8 +402,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
I can move this to it's own PR, and started to in https://github.com/nodejs/node/pull/9035 - but the problem is that the tests then fail without the other changes from that commit. And, then the second commit isn't actually tested without those changes.

So, I can break this up if you really want, but I don't think it makes sense to in this case.